.metadata/.plugins/org.eclipse.core.runtime/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```

Comment:
You should add `.metadata` folder to the `.gitignore` file. We don't need all these files.